src/framework/standard/args.rs

```diff
@@ -746,13 +746,27 @@ impl Args {
         &self.message[start..]
     }
 
+    /// Starting from the offset, return the remainder of available arguments.
+    ///
+    /// Returns `None` if there are no remaining arguments.
+    #[inline]
+    pub fn option_rest(&self) -> Option<&str> {
```

Comment:
I propose naming this a synonym of "rest":
```suggestion
    pub fn remains(&self) -> Option<&str> {
```